apps/site/src/app/apply/sections/Form/Form.tsx

```diff
@@ -38,11 +39,12 @@ export default function Form() {
 			<form
 				method="post"
 				className={`${styles.form} text-[#000000] w-8/12 flex flex-col items-center py-12 gap-10 z-1 max-[800px]:w-9/12 max-[400px]:w-11/12`}
-				action="/api/user/apply"
+				encType="multipart/form-data"
```

Comment:
Chore: I think we still need the action endpoint here?

Reply:
Oops, I thought I left it on merge